internal/cmd/run.go

```diff
@@ -317,3 +344,60 @@ func confirmExecution(cmd *cobra.Command, numTasks int, parallel bool, category
 
 	return nil
 }
+
+func promptForEnvVar(cmd *cobra.Command, ev CommandExportExtractMatch) (string, error) {
+	label := fmt.Sprintf("Set Environment Variable \" %s\"", ev.Key)
+	ip := prompt.InputParams{Label: label, PlaceHolder: ev.Value}
+	if ev.HasStringValue {
+		ip.Value = ev.Value
+	}
+	model := tui.NewStandaloneInputModel(ip, tui.MinimalKeyMap, tui.DefaultStyles)
+
+	finalModel, err := newProgram(cmd, model).Run()
+	if err != nil {
+		return "", err
+	}
+	val, ok := finalModel.(tui.StandaloneInputModel).Value()
+	if !ok {
+		return "", errors.New("canceled")
+	}
+	return val, nil
+}
+
+func replaceVarValue(ev CommandExportExtractMatch, newValue string) string {
+	parts := strings.SplitN(ev.Match, "=", 2)
+	replacedText := fmt.Sprintf("%s=%s", parts[0], newValue)
+	return replacedText
+}
+
+func getCommandExportExtractMatches(lines []string) []CommandExportExtractMatch {
+	test := regexp.MustCompile(exportExtractRegex)
+	result := []CommandExportExtractMatch{}
+
+	for i, line := range lines {
+		for _, match := range test.FindAllStringSubmatch(line, -1) {
+			e := match[0]
+
+			parts := strings.Split(strings.TrimSpace(e[len("export "):]), "=")
```

Comment:
```suggestion
			parts := strings.Split(strings.TrimSpace(e)[len("export "):], "=")
```

The javascript impl is subtly different 😉

Reply:
It was working lol I already change it